localhost/Dockerfile.dev

```diff
@@ -13,5 +13,8 @@ RUN curl -fsSL https://deb.nodesource.com/setup_16.x | bash
 RUN apt-get install -qq nodejs libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev \
                         libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
 
+# todo: need to extract this
+USER calitp
```

Comment:
Oops didn't mean to leave the todo in... but the point still stands that I want to use `$USER` if that is still visible from `Dockerfile`. Need to test if that works. Or chime in if you know that it works!

Reply:
> The environment variables set using ENV will persist when a container is run from the resulting image. 

From https://docs.docker.com/engine/reference/builder/#env